### ISSUE TITLE
Update the bonding cards

### ISSUE THREAD

**Issue by Giorgionocera** [bitsongofficial/sinfonia-ui#164]
The bonding cards need to be updated to add other information.
More specifically we need to add:
- the starting date for each incentive row;
- we can separate the currently released incentives from the upcoming ones, for example, by separating them through a line.

@giannnni, I need your support on this task. Can you help me find the best solution? It should be a nice idea to also add a design for this.

Current implementation:
![bonding-cards](https://user-images.githubusercontent.com/13519292/166301204-79cc6305-e2b5-4ddb-8a6c-9644dff290a3.jpg)

**Comment by giannnni** [comment]
Fixed as suggested.
<img width="1070" alt="Schermata 2022-05-06 alle 11 54 00" src="https://user-images.githubusercontent.com/92384659/167109671-99ec3e84-44b6-4662-b2b0-d3ddfc19e010.png">

**Comment by Giorgionocera** [comment]
Thank you @giannnni!
@DavideSegullo let's move to the implementation.

**Comment by Giorgionocera** [comment]
@DavideSegullo do you have any update on this?

**Comment by DavideSegullo** [comment]
@Giorgionocera  Yes, we don't have a behaviour or a component like that, so we should add it
